src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -0,0 +1,83 @@
+"""Authors: Szonja Weigl, Cody Baker."""
+from typing import Optional, List
+
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from ....utils import FolderPathType, get_schema_from_method_signature
+
+
+class OpenEphysLegacyRecordingInterface(BaseRecordingExtractorInterface):
+    """Primary data interface for converting legacy Open Ephys data (.continuous files).
+    Uses :py:class:`~spikeinterface.extractors.OpenEphysLegacyRecordingExtractor`."""
+
+    ExtractorName = "OpenEphysLegacyRecordingExtractor"
```

Comment:
Is this line needed?

Reply:
I would be interested to know as well - it's possible that this is needed in order for the lazy import to work, but also possible that the router triggers the correct sub-call at the lower level